### DISCUSSION TITLE
Migrating from Delta Tracker

### DISCUSSION THREAD

**Discussion by zjean** [ghostfolio/ghostfolio#3062] [Q&A]
Hi, thanks for this amazing tool!
Has anybody migrated from [Delta Investment Tracker](https://delta.app)? 
How did you approach this?
I am looking how to do this, preferably by not manually entering all my transactions in Ghostfolio 😁

**Comment by dtslvr** [comment]
Hi @zjean

Thank you for the feedback.

I'm not aware of such a converter. Maybe you can get inspired by similar scripts from the community [here](https://github.com/topics/ghostfolio).
